exercises/dnd-character/example.py

```diff
@@ -0,0 +1,24 @@
+import random
+import math
+
+
+class Character:
+    character = {}
+
+    def __init__(self):
+        self.character = {
+            'strength': self.ability(),
+            'dexterity': self.ability(),
+            'constitution': self.ability(),
+            'intelligence': self.ability(),
+            'wisdom': self.ability(),
+            'charisma': self.ability()
+            }
+        self.character['hitpoints'] = 10 + self.modifier(
+                    self.character['constitution'])
+
+    def ability(self):
+        return random.randint(3, 18)
```

Comment:
The intent of the exercise is "to roll 4 dice and sum the highest 3" to generate ability values. Technically, this produces the same result as `randint(3, 18)`... I'm stuck on:

- How essential this difference is in properly holding to the spirit of the exercise?
- How to enforce the "roll 4 dice" method

Reply:
> We can have the ability function return 3 values and sum them in an upper
function? Testing for that can verify if 3 values were returned.

I asked this question in the PR where this exercise was implemented. I really like the response I got: https://github.com/exercism/problem-specifications/pull/1397#issuecomment-443716867

To apply this statement to this PR: leave the tests as-is, change the example implementation to match the intent.

```suggestion
		dice_rolls = sorted([random.randint(1, 6) for _ in range(4)])
		return sum(dice_rolls[1:])
```

Reply:
Changed the ability method to better exemplify the spirit of the exercise.